src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.targets

```diff
@@ -37,7 +37,8 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
 
   <PropertyGroup>
     <TargetProfile Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' " >mscorlib</TargetProfile>
-    <TargetProfile Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework' " >netcore</TargetProfile>
+    <TargetProfile Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework' AND ($(TargetFramework.StartsWith(netcoreapp1.)) OR $(TargetFramework.StartsWith(netstandard1.)))" >netcore</TargetProfile>
```

Comment:
I just looked at cases down the line and yes I think those should be modified too. TargetFramework is not even canonical - net46 and net4.6 both map to the same framework. So some of the checks below will be broken even for the regular cases. In general, we should avoid interpreting the targetframework property directly.

Note that if you need to do a bunch of version comparisons, you can do something like [this ](https://github.com/dotnet/sdk/blob/79d2f9e93433ff198516f2defccf9fe37c20a887/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.BeforeCommon.targets#L87) and then msbuild lets you do direct integer comparisons - _TargetFrameworkVersionWithoutV > 2.0